modules/social_features/social_group/social_group.install

```diff
@@ -654,3 +654,17 @@ function social_group_update_8907() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add translations support for views with groups.
+ */
+function social_group_update_8908() {
```

Comment:
@nechai As we agreed better to start hook update with the target OS version, so it will be `10201`

Reply:
@ribel Resolved (changed destination branch to 10.3.x)

Reply:
@nechai It's good that you change the destination branch to `10.3.x`, but since we target this PR to milestone `10.2.1`, I think it still better to use hook update `10201` rather than `10301`.

Reply:
@ribel Renamed hooks regarding milestone number